### DISCUSSION TITLE
Maybe techical issue with waitlists?

### DISCUSSION THREAD

**Discussion by TheBinaryGuy** [community/community#4494] [Codespaces] [answered]
I've been on the waitlist for Codespaces forever, I signed up within the couple of days of it's launch and it still shows that I'm on the waitlist. All my colleagues who signed up recently got in, and that also within couple of days.

Same for Copilot, most people I know got in within 24-48 hours, while I signed up on the first day and still am on the waitlist.

Maybe there's a technical issue with how the waitlist behaves? Should I try with a fresh account?

**Comment by qinyihao** [comment]
I also encountered this problem. I'd like the github officials to answer this question

**Reply by TheBinaryGuy** [reply]
Let's see what the reply is, I'll also create a support ticket, if anything comes up, I'll keep you informed.

**Reply by azartheen** [reply]
i signup copilot but it almost more than 5 days I am still on the waitlist!... can i know the reason why i can't get access

**Comment by thanhcsf** [comment]
I think you can send a help request to Github, maybe Github's staff will check the problem and support your request in a more specific and reasonable way.

**Reply by qinyihao** [reply]
I've sent several tickets to the github staff, and all they answer me is:

![image](https://user-images.githubusercontent.com/40647091/124389247-e07a5800-dd18-11eb-8065-d7d5a875fe3f.png)

**Reply by TheBinaryGuy** [reply]
Yep, I'm sending them a support request.

**Reply by KoolenDasheppi** [reply]
> I've sent several tickets to the github staff, and all they answer me is:
> 
> ![image](https://user-images.githubusercontent.com/40647091/124389247-e07a5800-dd18-11eb-8065-d7d5a875fe3f.png)

I had the same exact reply as well when I contacted them. This whole situation is very frustrating.

**Comment by Zerotask** [comment]
It's called a **Wait**list. This means you have to wait like everyone.
Creating discussions and send help requests won't increase your chances, it will rather decrease it.

So just stay calm and wait.

**Comment by tanmayeekamath** [comment, marked as answer]
We are aware that you would like to access GitHub Codespaces (and we're very excited to share it with you!) but we're being cautious with our rollout right now so I ask that you hang tight. We'll add more folks as soon as we can! Thanks for your patience. :)

**Reply by TheBinaryGuy** [reply]
No worries, I'll wait. Just wanted to make sure that it wasn't a technical issue or something with my account.

Got the access to Copilot! 👍

**Comment by ajsirota** [comment]
With Codespaces rolling out to paid orgs today, is there an ETA for it becoming available to individuals? Even just an update on my position in the waitlist would be awesome.

**Reply by tanmayeekamath** [reply]
We’ll share more information about how individuals will be able to use Codespaces in the future. Individuals in the existing beta will continue to have access to Codespaces and will not be charged. Regarding the waitlist, we're being cautious with our rollout right now considering our focus on GA for teams/orgs so we ask that you hang tight. We'll add more folks as soon as we can! Thanks for your patience.
